app/lib/test_applications.rb

```diff
@@ -179,6 +162,8 @@ def create_application(recruitment_cycle_year:, states:, courses_to_apply_to:, a
         FactoryBot.create(:ucas_match, application_form: @application_form) if rand < 0.5
       end
 
+      application_choices.each { |a| a.update!(updated_at: Time.zone.now, audit_comment: 'This application was automatically generated') }
```

Comment:
Just to confirm the audit functionality doesn't support bulk operations with an `audit_comment`?

Reply:
I'll double check this. The reason it's like this is because application choices is an array rather than a relation. Maybe I can do the update via `@application_form` though